src/main/resources/jenkins/branch/MultiBranchProject/configure-common.jelly

```diff
@@ -23,17 +23,11 @@
  -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:p="/lib/hudson/project" xmlns:this="this">
-  <p:config-disableBuild/>
```

Comment:
Is there matching pull requests to add these back in?

it would seem possible to check if the MBP is instantiating subtypes of Project/AbstractProject and use a `<j:if>` to include them. Otherwise we should bump the major version of the branch api and flag compatability issues via the hpi plugin confif